forem/forem
# Post bootstrap changes - remove headings from admin.scss

## What type of PR is this? (check all applicable)


## Description

This PR removes all the headings that we are overriding in admin.scss file.

This PR uses following guidelines mainly (not fully).
| Heading  | Crayons Style |
| ------------- | ------------- |
| h1 | crayons-title |
| h2 | crayons-subtitle-1 |
| h3 | crayons-subtitle-3 |
| h4 | crayons-subtitle-4 |

## Related Tickets & Documents



- Closes #19018

## QA Instructions, Screenshots, Recordings

There are no major changes in UI but the code consistency and styles have been changed. 

**How can this best tested?**

- you can checkout in this branch, run the code locally as admin and check all admin pages/features and lookout for any weird UI.
- you can test it on https://canary1.forem.lol/admin/


### UI accessibility concerns?

Improves accessibility as there is exactly 1 h1 tag in all pages. And also the headings have a order of h1-h6, i.e. there won't be a page where we use h1 and then directly use h5 skipping h2,h3,h4 etc.

## Added/updated tests?


## [Forem core team only] How will this change be communicated?


## [optional] Are there any post deployment tasks we need to perform?

@maestromac We will need to deploy this on Canary1 or Canary2 to test.
